test/utils/setUtilization.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  MockTrueCurrency,
+  StkTruToken,
+  TrueFiPool2,
+  TrueRatingAgencyV2,
+  TrueLender2,
+  LoanFactory2,
+} from 'contracts'
+import { BigNumber, Wallet } from 'ethers'
+import { MockProvider } from 'ethereum-waffle'
+import { DAY } from './constants'
+import { createApprovedLoan as _createApprovedLoan } from './createLoan'
+
+// When setting utilization above 30 percent
+// slightly increases pool value
+export const setUtilization = async (
+  provider: MockProvider,
+  rater: TrueRatingAgencyV2,
+  tru: MockTrueCurrency,
+  stkTru: StkTruToken,
+  tusd: MockTrueCurrency,
+  loanFactory: LoanFactory2,
+  borrower1: Wallet,
+  borrower2: Wallet,
+  lender: TrueLender2,
+  owner: Wallet,
+  pool: TrueFiPool2,
+  utilization: number,
+) => {
+  if (utilization === 0) {
+    return
+  }
+
+  const createApprovedLoan = async (borrower, amount) => _createApprovedLoan(
+    rater, tru, stkTru,
+    loanFactory, borrower, pool,
+    amount, DAY, 0,
+    owner, provider,
+  )
+
+  const poolValue = await pool.poolValue()
+  const utilizationAmount = poolValue.mul(utilization).div(100)
+  if (utilization <= 15) {
+    const loan = await createApprovedLoan(borrower1, utilizationAmount)
+    await lender.connect(borrower1).fund(loan.address)
+    return
+  }
+  if (utilization <= 30) {
```

Comment:
if else?

Reply:
There is no need (unless it might improve readability which I doubt). Notice early return statement in previous *if*.